### PR TITLE
fix: markdown editor preview

### PR DESCRIPTION
**Pull request by MohammedAl-Rowad**
### Fixing the preview in the markdown editor

Current behaviour:

HTML code snippet not rendering + no syntax highlighting:

https://user-images.githubusercontent.com/38977667/173547249-28ddaf89-60ff-44e2-83a8-a633b8a6d3de.mp4

Code not previewed correctly:

https://user-images.githubusercontent.com/38977667/173547319-fc2201de-5c1c-493f-9a1d-fe929e94fa0d.mp4

After the fix:

https://user-images.githubusercontent.com/38977667/173547420-e77b5a0a-43e9-4096-b91d-c4bc82587060.mp4

### CODE REVIEW

File: packages/markdown/src/components/MarkdownPreview.tsx [hunk 1]
```diff
@@ -196,27 +209,40 @@ export const MarkdownPreview = React.memo((props: MarkdownPreviewProps) => {
...
-  // https://github.com/cure53/DOMPurify/blob/main/src/tags.js#L3-L121
-  const cleanHTML = React.useMemo(() => {
-    return DOMPurify.sanitize(props.value);
-  }, [props.value]);
-
   return (
     <div className={className} data-test-id="markdown-preview">
       <Markdown
         options={{
+          createElement(type, props, children) {
```
[the remark this answers is not in the document]
Comment (MohammedAl-Rowad) [reply packages/markdown/src/components/MarkdownPreview.tsx:216]: Maybe we should switch to [react-markdown](https://github.com/remarkjs/react-markdown#when-should-i-use-this)

looks to me that [markdown-to-jsx](https://github.com/probablyup/markdown-to-jsx) is not intended to display user input data, maybe a package for internal usage only 🤷

[the remark this answers is not in the document]
Comment (z0al) [reply packages/markdown/src/components/MarkdownPreview.tsx:216]: I'm not sure if using `react-markdown` will replace the need for `dompurify` but I agree `markdown-to-jsx` has plenty of issues and we should probably switch to something more mature.

[the remark this answers is not in the document]
Comment (MohammedAl-Rowad) [reply packages/markdown/src/components/MarkdownPreview.tsx:216]: I'll close this PR.

I just referenced it in some places

[the remark this answers is not in the document]
Comment (denkristoffer) [reply packages/markdown/src/components/MarkdownPreview.tsx:216]: My two cents here are that going with something by the remark organisation is probably the safest bet 👍

**Pull request closed without merging**